Automattic/wp-calypso
# Update/circular progress bar props

#### Testing/Review estimation

Testing: short
Review: short

#### Proposed Changes

Update the Circular progress bar component to use a `size` prop (changing the size of the component) and `enableMobileScaling` prop to apply mobile view port styling.

#### Testing Instructions



* Checkout this branch
* Create a new launchpad site or use an existing site.
* Navigate to the launchpad screen and verify the Circular Progressbar matches production.

<img width="310" alt="image" src="https://user-images.githubusercontent.com/10482592/215583677-ad8aad33-a268-46fa-9725-8fb5b9546570.png">


#### Pre-merge Checklist





Related to #
